src/_C014.ino

```diff
@@ -0,0 +1,686 @@
+                  cmd += (deviceNr+1); // set the device Number
+                  cmd += F(",");
+                  cmd += (valueNr+1); // set the value Number
+                  cmd += F(",");
+                  cmd += event->String2; // expect float as payload!
+                  validTopic = true;
+                }
+              }
+            }
+
+            if (validTopic) {
+              parseCommandString(&TempEvent, cmd);
+              log+=F(" cmd: ");
+              log+=cmd;
+              addLog(LOG_LEVEL_INFO, log+ F(" OK"));
+            } else {
+              addLog(LOG_LEVEL_INFO, log+ F(" ERROR"));
+            }
+
+            // Feedback should not be sent here because here it will only confirm sucessfull retreval of the message!
+            // CPlugin_014_sendMQTTmsg(pubname, event->String2.c_str(), errorCounter);
+
+          }
+
+          if (validTopic) {
+            log=F("C014 : Exec > ");
+            // in case of event, store to buffer and return...
+            String command = parseString(cmd, 1);
+            if (command == F("event"))
+            {
+              log+=F("Event!");
+              eventBuffer = cmd.substring(6);
+            } else {
+              log=F("C014 : PluginCall:");
+              if (!PluginCall(PLUGIN_WRITE, &TempEvent, cmd)) {
+                remoteConfig(&TempEvent, cmd);
+                log +=F(" FALSE! remoteConfig?");
+              } else {
+                log +=F(" OK!");
+              }
+              addLog(LOG_LEVEL_INFO, log);
+            }
+
+          } else TempEvent.Source = 0; // something whent wrong
+
+        }
+        break;
+      }
+
+    case CPLUGIN_PROTOCOL_SEND:
+      {
+        MakeControllerSettings(ControllerSettings);
+        LoadControllerSettings(event->ControllerIndex, ControllerSettings);
+        if (!ControllerSettings.checkHostReachable(true)) {
+            success = false;
+            break;
+        }
+        statusLED(true);
+
+        if (ExtraTaskSettings.TaskIndex != event->TaskIndex)
+          PluginCall(PLUGIN_GET_DEVICEVALUENAMES, event, dummyString);
+
+        String pubname = ControllerSettings.Publish;
+        parseControllerVariables(pubname, event, false);
+
+        String value = "";
+        // byte DeviceIndex = getDeviceIndex(Settings.TaskDeviceNumber[event->TaskIndex]);
+        byte valueCount = getValueCountFromSensorType(event->sensorType);
+        for (byte x = 0; x < valueCount; x++)
+        {
+          String tmppubname = pubname;
+          tmppubname.replace(F("%valname%"), ExtraTaskSettings.TaskDeviceValueNames[x]);
+          value = formatUserVarNoCheck(event, x);
+
+          MQTTpublish(event->ControllerIndex, tmppubname.c_str(), value.c_str(), Settings.MQTTRetainFlag);
+
+          String log = F("C014 : Sent to ");
+          log += tmppubname;
+          log += ' ';
+          log += value;
+          addLog(LOG_LEVEL_DEBUG, log);
```

Comment:
Please wrap this log string creation in a check for `logLevelActiveFor`

Reply:
done